setup.py

```diff
@@ -21,7 +21,7 @@
       python_requires="~=3.8",
       install_requires=[
           "ply>=3.4",
-          "pycparser>=2.18,<=2.20",
+          "pycparser==2.21",
```

Comment:
Will this include 2.21.1?

Reply:
It wouldn't, looks like I should use ~= 2.21 instead:
https://peps.python.org/pep-0440/#compatible-release